bin/s2nd.c

```diff
@@ -185,6 +187,11 @@ void usage()
     fprintf(stderr, "    Location of key file used for encryption and decryption of session ticket.\n");
     fprintf(stderr, "  -T,--no-session-ticket\n");
     fprintf(stderr, "    Disable session ticket for resumption.\n");
+    fprintf(stderr, "  --serialize-out [file path]\n");
+    fprintf(stderr, "    Path to a file where a serialized connection can be stored.\n"
+                    "    Note that this feature is intended to be used with our integration test framework and therefore is not expected to work with s2nd alone.\n");
+    fprintf(stderr, "  --deserialize-in [file path]\n");
+    fprintf(stderr, "    Path to a file where a serialized connection lives. Will be used to skip the handshake and start sending encrypted data.\n");
```

Comment:
Okay maybe this is an awful suggestion, but: if we don't want anyone to use this outside of the integration tests, then maybe we shouldn't print it as part of the help message? :/